setup.py

```diff
@@ -24,24 +24,27 @@ def exec_file(names):
 
 
 setup(
-    name="matrix_client",
-    version=exec_file(("matrix_client", "__init__.py",))["__version__"],
-    description="Client-Server SDK for Matrix",
-    long_description=read_file(("README.rst",)),
-    author="matrix.org",
-    author_email="team@matrix.org",
-    url="https://github.com/matrix-org/matrix-python-sdk",
+    name='matrix_client',
+    version=exec_file(('matrix_client', '__init__.py',))['__version__'],
+    description='Client-Server SDK for Matrix',
+    long_description=read_file(('README.rst')),
+    author='The Matrix Developers (matrix.org)',
```

Comment:
Only thing I want to add is could you make it 'Matrix.org team' as:
* Developers is a little vague, and technically speaking the SDK is still a core matrix project.
* The site refers to the team as 'Matrix.org team' so I'd like to keep it consistent.